frontend/package.json

```diff
@@ -43,6 +43,7 @@
     "react-redux": "^8.0.5",
     "react-router-dom": "6.2.1",
     "react-scripts": "^5.0.0",
+    "react-select-event": "^5.5.1",
```

Comment:
⚠️ leveraged https://testing-library.com/docs/ecosystem-react-select-event since I was having a difficult time getting our combobox interactions working as intended with these tests

Even though it was meant to be used with `react-select` components, the `select` command worked with uswds comboBoxes...

Unfortunately, the `clearFirst` and `clearAll` command do not work as the implementation assumes there is an `svg` which I don't think uswds comboBoxes use (https://github.com/romgain/react-select-event/blob/e70973fec5278340b06fc3a6766a464c3de8b063/src/index.ts#L142C1-L160)